webnn/resources/utils.js

```diff
@@ -277,6 +277,14 @@ const PrecisionMetrics = {
   min: {ULP: {float32: 0, float16: 0}},
   pow: {ULP: {float32: 32, float16: 2}},
   // End Element-wise binary operations
+  // Begin Element-wise logical operations
+  equal: {ULP: {uint8: 0}},
```

Comment:
For logical operations, no mater input data type is float32 or float16 and etc. , the output type should be **uint8**.
And here **uint8** is align with expected data type in json file, likes

```
"expected": {
  "name": "output",
  "shape": [2, 1, 4, 1, 3],
  "data": [
    0,
    0,
    0,
    1,
    0,
    ...
    0
  ],
  "type": "uint8"
}
```

@fdwr Is it OK for this addition code? Thanks.